bk-api/test_bk_api.py

```diff
@@ -139,6 +139,30 @@ def test_log_insert_fail(client):
     assert 'error'  in result
 
 
+def test_vsn_insert(client):
+    #Check that the initial value is null
+    rv = client.get(f'/state/0000000000000001')
+    result_before_vsn = rv.get_json()
+    d_before_vsn = result_before_vsn["data"]["vsn"]
+    assert 'error' not in result_before_vsn
+    assert 'data' in result_before_vsn
+    assert None == d_before_vsn
+    #Check that we can populate vsn
+    data = {"vsn": True}
+    rv = client.post('/node/0000000000000001', data = json.dumps(data))
+    result = rv.get_json()
+    assert "success" in result
+    #Check that the db is updated with the new value
+    gt_value = "TEST-minimal"
+    rv = client.get(f'/state/0000000000000001')
+    result = rv.get_json()
+    print(result)
+    assert 'error' not in result
+    assert 'data' in result
+
+    d_after_vsn = result["data"]["vsn"]
```

Comment:
can u move this to right after the `rv.get_json()` call like you did on line 146?

its easier to compare the "before" and "after"

Reply:
That was my accident. Made the change just now. 
I need the assert statements before I pass in a key into the dictionary. If not I will error out before I get to the assert.